scipy/optimize/_differentialevolution.py

```diff
@@ -708,6 +717,12 @@ def maplike_for_vectorized_func(func, x):
         # _unscale_parameter. This is an optimization
         self.__scale_arg1 = 0.5 * (self.limits[0] + self.limits[1])
         self.__scale_arg2 = np.fabs(self.limits[0] - self.limits[1])
```

Comment:
since `__scale_arg2` is no longer used, I would have suggested omitting it and just defining `__recip_scale_arg2`. Alternatively, this logic could go in `_unscale_parameters`, since that's the only place the quantity is used.
But you're the primary maintainer of this feature, so I suppose you prefer it this way.

Reply:
`__scale_arg2` is still used in the `_scale_parameters` method, so it's still necessary to keep it around. This method is heavily used in the minimisation loop.

Reply:
Oops I did a search and I didn't think it was used. My mistake.